lib/src/pages/leaderboard/leader_board_page.dart

```diff
@@ -38,14 +46,24 @@ class _LeaderBoardPage extends State<LeaderBoardPage> {
         centerTitle: true,
       ),
       body: Center(
-        child: Column(
-          children: const <Widget>[
-            Period(),
-            FirstThree(),
-            Expanded(
-              child: ScoreList(),
-            ),
-          ],
+        child: FutureBuilder(
+          future: ScoreService.instance.findAllByGameUid(_game.uid).then(
+                (value) => {
+                  _userScore = value.sublist(numberTop),
+                  _threeTopUser = value.sublist(numberFirstElement, numberTop),
+                },
+              ),
+          builder: (context, userSoreSnap) {
```

Comment:
ich finde Abkürzungen immer problematisch und empfehle dir, darauf zu verzichten. Lieber gleich userScoreSnapshot, dann lässt sich der Code insgesamt auch flüssiger lesen. 
Im Buch schreibt ja auch keiner Und dann stieg P. in sein Auto und fuhr auf die Landstr. :D